docs/modules/ROOT/pages/references/architecture/control-api-billing-entity.adoc

```diff
@@ -0,0 +1,99 @@
+= {controlapi}: BillingEntity
+
+TIP: This resource implements the xref:references/functional-requirements/portal.adoc#_feature_manage_billing_information["Manage Billing Information"] features.
+
+It is required that billing data can be modified by the billing entity owner in the portal and by our accounting team in the ERP.
```

Comment:
I am missing a general description what a BillingEntity is supposed to be, what its purpose is. That lack of that information, makes this a bit hard to read. Yes, there is the entry in the glossary (not linked). But here I would like to see a more elaborate description.

Reply:
I tried to describe it a bit more but it still feels thin. Maybe you have some ideas too?